package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
+    "ios": "nx run-ios",
+    "android": "nx run-android",
```

Comment:
should this not be `npx nx...`

i.e 
```json
   "ios": "npx nx run-ios",
    "android": "npx nx run-android",
 ```

Reply:
That will run `nx` from `node_modules` over a global one. But yes, there probably are instances where conflicts could arise. But we should never really have `npx` as part of package scripts.